Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/DiagnosticsTab/DiagnosticFilters.tsx

```diff
@@ -0,0 +1,156 @@
+      color: isChecked ? SharedColors.red10 : NeutralColors.gray90,
+      fontSize: FontSizes.size16,
+      marginRight: '8px',
+    },
+  };
+
+  if (severityType === DiagnosticSeverity.Warning) {
+    return {
+      ...baseStyle,
+      icon: {
+        ...baseStyle.icon,
+        color: isChecked ? '#F4BD00' : NeutralColors.gray90,
+      },
+    };
+  } else {
+    return baseStyle;
+  }
+};
+
+type DiagnosticsFiltersProps = {
+  showErrors: boolean;
+  projectsToFilter: string[];
+  showWarnings: boolean;
+  setWarningFilter: (value: boolean) => void;
+  setErrorFilter: (value: boolean) => void;
+  setProjectsToFilter: (projects: string[]) => void;
+  errorCount: number;
+  warningCount: number;
+};
+
+export const DiagnosticsFilters: React.FC<DiagnosticsFiltersProps> = (props) => {
+  const {
+    showErrors,
+    showWarnings,
+    setWarningFilter,
+    setErrorFilter,
+    setProjectsToFilter,
+    projectsToFilter,
+    errorCount,
+    warningCount,
+  } = props;
+  const projects = useRecoilValue(outputsDebugPanelSelector);
+  const [projectSelectorOptions, setProjectSelectorOptions] = useState<IDropdownOption[]>([]);
+  const rootBotProjectId = useRecoilValue(rootBotProjectIdSelector);
+
+  useEffect(() => {
+    if (rootBotProjectId) {
+      setProjectsToFilter([rootBotProjectId]);
+    }
+  }, [rootBotProjectId]);
+
+  useEffect(() => {
+    const projectOptions = projects.map(({ projectId, botName }) => {
+      return {
+        key: projectId,
+        text: botName,
+      };
+    });
+    setProjectSelectorOptions(projectOptions);
+  }, [projects]);
+
+  if (!rootBotProjectId) {
+    return null;
+  }
+
+  return (
+    <div
+      css={{
+        display: 'flex',
+        flex: '1 1 auto',
+        width: '100%',
+      }}
+    >
+      <DefaultButton
+        allowDisabledFocus
+        checked={showErrors}
+        iconProps={{ iconName: 'StatusErrorFull' }}
+        styles={getSeverityButtonStyle(DiagnosticSeverity.Error, showErrors)}
+        onClick={() => {
+          setErrorFilter(!showErrors);
+        }}
+      >
+        <span style={severityTextStyle}>{formatMessage('Errors')}</span>
+        <span style={countFontWeight}>{errorCount}</span>
+      </DefaultButton>
+      <DefaultButton
+        allowDisabledFocus
+        toggle
+        checked={showWarnings}
+        iconProps={{ iconName: 'WarningSolid' }}
+        styles={getSeverityButtonStyle(DiagnosticSeverity.Warning, showWarnings)}
+        onClick={() => {
+          setWarningFilter(!showWarnings);
+        }}
+      >
+        <span style={severityTextStyle}>{formatMessage('Warnings')}</span>
+        <span style={countFontWeight}>{warningCount}</span>
+      </DefaultButton>
+      <div
+        css={{
+          marginLeft: 'auto !important',
+        }}
+      >
+        <DropdownWithAllOption
+          dropdownOptions={projectSelectorOptions}
+          optionAllKey={optionAllKey}
+          optionAllText={formatMessage('All bots')}
```

Comment:
"all **bots**" then "select **projects**"
I think it's better if we stick with one of the two terms

Reply:
Aah. Yep.